### ISSUE TITLE
ipyrad conda environment: sharde library

### ISSUE THREAD

**Issue by ARW-UBT** [dereneaton/ipyrad#427]
Hello, 
I have installed ipyrad as a conda environment, but I receive the following error message when following the advanced zutorial: 

Encountered an Error.
Message: IPyradError: b'/opt/miniconda3/envs/ipyrad/bin/samtools: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory\n'

Any idea how I could fix that error?

Best regards

**Comment by isaacovercast** [comment]
Hello, This is an issue with your conda environment. Simplest thing to try is just wikping your miniconda3 directory and reinstalling conda and create a new environment from scratch.

Other ideas for solving this are here: https://github.com/merenlab/anvio/issues/1479

**Comment by alexkrohn** [comment]
Hi @isaacovercast,

This error seems to persist in the latest release of ipyrad. Resintalling samtools 1.7 doesn't seem to fix it for me, and installing samtools > 1.7 throws conflicts for a number of packages (zlib, openssl, libgcc-ng).

Here's what I've done:
```
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
bash Miniconda3-latest-Linux-x86_64.sh
bash
conda create -n ipyrad python=3.9
conda activate ipyrad
conda install ipyrad -c conda-forge -c bioconda

samtools
# samtools: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory

conda install -c bioconda samtools=1.7 --force-reinstall
samtools
# samtools: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
``` 

I'm running ipyrad v0.9.82 on Python 3.9.10 on Ubuntu 20.04

**Comment by isaacovercast** [comment]
@alexkrohn,
Did you figure this out? I just did a clean install of ipyrad and had no
trouble at all. Is this still happening for you?

**Comment by alexkrohn** [comment]
Hey @isaacovercast. I deleted my issue because I resolved it using the tactics in merenlab/anvio#1479. This issue arose on a fresh install of Ubuntu 20.04, with a fresh install of miniconda and ipyrad v0.9.82. 

First I had to install libssl1.0
```
sudo nano /etc/apt/sources.list
add deb http://security.ubuntu.com/ubuntu bionic-security main
sudo apt update && apt-cache policy libssl1.0-dev
sudo apt-get install libssl1.0-dev
```
Then manually re-install samtools 1.7 in the ipyrad environment
```
conda activate ipyrad
conda uninstall -c bioconda samtools
conda install -c bioconda samtools=1.7
```

That didn't actually resolve all my issues, as I had to address [the vsearch version issue](https://github.com/dereneaton/ipyrad/issues/469) too. So, after running `conda install -c bioconda vsearch=2.19`, I was good to go.

Thanks for checking in. It seems resolved now.
